Dockerfile

```diff
@@ -1,7 +1,9 @@
 FROM ruby:2.7-slim-bullseye
 
 RUN apt-get update -y
-RUN apt-get install -y git zip
+RUN apt-get install -y git zip python3.9
+RUN curl -sSL https://install.python-poetry.org | python3 - --version 1.1.13
+ENV PATH="${PATH}:/root/.poetry/bin"
```

Comment:
Have you tried to build this image locally? I couldn't get it to work as is, and have tinkered around with the following which seems to build and run. There's a few dependencies required, e.g. curl, pip etc.. Also python 3 couldn't be found, so I specified 3.9 when using python. poetry couldn't be found, so I was explicit about the install directory. 

:

```docker
RUN apt-get update -y
RUN apt-get install -y git zip curl python3.9 python3-venv python3-pip
ENV POETRY_HOME=/etc/poetry
RUN curl -sSL https://install.python-poetry.org | python3.9 - --version 1.1.13
ENV PATH="${PATH}:/${POETRY_HOME}/bin"
```